app/app.go

```diff
@@ -811,17 +807,30 @@ func (app *App) NewProposalContext(header tmproto.Header) sdk.Context {
 	return ctx
 }
 
-// TimeoutCommit returns the timeout commit duration to be used on the next block.
-// It returns the user specified value as overridden by the --timeout-commit flag, otherwise
-// the default timeout commit value for the current app version.
-func (app *App) TimeoutCommit() time.Duration {
-	if app.timeoutCommit != 0 {
-		return app.timeoutCommit
+// TimeoutHeightDelay returns the timeout commit duration to be used on the next block.
+// It returns the user specified value as overridden by the --block-time flag, otherwise
+// the default timeout height delay value for the current app version.
+func (app *App) TimeoutHeightDelay() time.Duration {
+	if app.blockTime != 0 {
+		return app.blockTime
 	}
 
 	return appconsts.TimeoutCommit
 }
 
+func (app *App) TimeoutInfo() abci.TimeoutInfo {
+	return abci.TimeoutInfo{
+		TimeoutPropose:          appconsts.TimeoutPropose,
+		TimeoutProposeDelta:     appconsts.TimeoutProposeDelta,
+		TimeoutCommit:           appconsts.TimeoutCommit,
+		TimeoutPrevote:          appconsts.TimeoutPrevote,
+		TimeoutPrevoteDelta:     appconsts.TimeoutPrevoteDelta,
+		TimeoutPrecommit:        appconsts.TimeoutPrecommit,
+		TimeoutPrecommitDelta:   appconsts.TimeoutPrecommitDelta,
+		DelayedPrecommitTimeout: app.blockTime,
```

Comment:
I'm a bit confused here. The DelayedPrecommitTimeout isn't necessary the block time, it's the minimum time before nodes precommit

Reply:
cool sounds good did in https://github.com/celestiaorg/celestia-app/pull/5639/commits/b8e2364290b10e071a2070063c6d870a63eb5bf9

I think the reasoning to try and change the names here was misguided, and instead of tried to push for renaming that constant in core. 

1) all other timeouts use a different format `Timeout...` instead of `...Timeout`
2) We already have TimeoutCommit, TimeoutPrecommit, TimeoutCommitDelta. If we added TimeoutDelayedPrecommit, that would be okay, but idk how someone not deeply familar with the changes is supposed to understand that they need to change this time in order to adjust the block time